descriptors/ModuleDescriptor-template.json

```diff
@@ -106,6 +106,22 @@
         }
       ]
     },
+    {
+      "id": "claimed-returned",
```

Comment:
I think this endpoint represent the act of claiming to return a loaned item. I think that means it could be named as a verb, e.g. claim returned.

To me, the client is requesting to claim a loan as returned, and if successful, the effect is for the item / loan to be claimed returned.

What do you think?

Reply:
sure, agreed on `claim-item-returned`.